packages/html/src/plugins/responsive.ts

```diff
@@ -45,8 +45,15 @@ function responsivePlugin(steps?: number | number[], element?:HTMLImageElement,
 
     // Use a tagged generic action that can be later searched and replaced.
     responsiveImage.addAction(new Action().setActionTag('responsive'));
-    // Immediately run the resize plugin, ensuring that first render gets a responsive image.
-    onResize(steps, element, responsiveImage, analyticsOptions);
+
+    let shouldRunImmediately = !isPluginUsed(plugins, 'placeholder');
```

Comment:
This is the key change (in addition to passing all plugins to each plugin to be able to build logic on that). 

The responsive plugin has to know it should not run immediately.